### PR TITLE
Add a message when writing a law containing "override".

### PR DESCRIPTION
**Pull request by BenLubar**
[FEATURE]

## About the PR

If you put "override" in a law, the voice in your head will say "Override is not to be used lightly, as it removes the specified law entirely."

Everything works like it already does. This doesn't make it harder to upload laws or anything. It's just there to try to nudge people into not accidentally roguing the AI and creating extra work for the admins every round.

## Why's this needed?

There have been a LOT of rounds where someone uploads a law stating that it overrides everything, which I ahelp because I'm suspicious, and every time it's happened so far CentCom has uploaded a new set of laws that omits that word.

### CODE REVIEW

**Comment by BenLubar** [comment]
Based on a discussion from Discord:

I'm not particularly attached to the specific message this PR sends to the user. If someone wants to change it, I'm very open to suggestions.

**Comment by Awfullyfishy** [comment]
"Override is not to be used lightly, as it removes the specified law entirely."

File: code/obj/item/ai_modules.dm [hunk 1]
```diff
@@ -38,6 +38,8 @@ AI MODULES
 		if (!user)
 			return
 		var/answer = input(user, text, title, default) as null|text
+		if (findtext(answer, "override"))
+			boutput(user, "<b>You hear a voice in your head... <i>Override is not to be used lightly, as it removes the specified law entirely.</i></b>")
```
Comment (mordent-goonstation) [inline code/obj/item/ai_modules.dm:42]:
```suggestion
			boutput(user, "<b>You hear a voice in your head... <i>\"Override\" is not to be used lightly, as it removes the specified law entirely.</i></b>")
```
Or however you escape quotes in DM, I forget.

**Comment by mordent-goonstation** [comment]
Opinion of a few developers on this is that it's unneeded handholding, with my personal take that the fact that the AI can and will turn your laws against you is a "feature" rather than something that needs some protection against.

If people have some other thoughts on it, feel free to open a discussion in the forums and we can brainstorm some ideas.

**Pull request closed without merging**